CONTRIBUTORS.md

```diff
@@ -0,0 +1,14 @@
+# Contributors
+
+Thank you to all the people who have already contributed to git-point!
```

Comment:
As @lex111 said, the official name is "GitPoint", so I'd say we should probably use that. I'm actually not sure why there's a dash in the repository name 😂

Reply:
Done!

Reply:
@lex111 I always use `kebab-case` for repository names, but I really don't know why I didn't just name it `gitpoint` :thinking: